kubeflow/modeldb/prototypes/modeldb.jsonnet

```diff
@@ -0,0 +1,17 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.modeldb
+// @description Model management service for KubeFlow
+// @shortDescription Model management
+// @param name string Name to give to each of the components
+// @optionalParam modeldbImage string mitdbg/modeldb-backend:latest The image for modeldb
```

Comment:
Can we pin to immutable tags so that the deployment won't suddenly break because the latest tag was moved?